examples/jsm/objects/ReflectorForSSRPass.js

```diff
@@ -40,33 +41,35 @@ var Reflector = function ( geometry, options ) {
 	//
 
 	scope.needsUpdate = false;
-	scope.maxDistance = Reflector.ReflectorShader.uniforms.maxDistance.value
-	scope.opacity = Reflector.ReflectorShader.uniforms.opacity.value
-
-  scope._isDistanceAttenuation = Reflector.ReflectorShader.defines.isDistanceAttenuation
-  Object.defineProperty(scope, 'isDistanceAttenuation', {
-    get() {
-      return scope._isDistanceAttenuation
-    },
-    set(val) {
-      if (scope._isDistanceAttenuation === val) return
-      scope._isDistanceAttenuation = val
-      scope.material.defines.isDistanceAttenuation = val
-      scope.material.needsUpdate = true
-    }
+	scope.maxDistance = ReflectorForSSRPass.ReflectorShader.uniforms.maxDistance.value
+	scope.opacity = ReflectorForSSRPass.ReflectorShader.uniforms.opacity.value
+	scope.worldYBias = options.worldYBias || 0;
+	scope.resolution = options.resolution || new Vector2( window.innerWidth, window.innerHeight );
+
+	scope._isDistanceAttenuation = ReflectorForSSRPass.ReflectorShader.defines.isDistanceAttenuation
+	Object.defineProperty(scope, 'isDistanceAttenuation', {
```

Comment:
How is `isDistanceAttenuation` used?

Reply:
Hello Mr.doob. This is the uncleaned code left over from SSRPass. I think refraction should not need the distanceAttenuation feature. I started this SSRrPass just today, at present, it is only the stage that can be seen and want to share first. Including many other places, I'll clean up later.

Reply:
If on, the reflection's opacity will has fade effect, otherwise all same opacity.
And because of I used it as `defines`, so don't want set it and update material in render loop, thus used `defineProperty` to watch it's change state.

isDistanceAttenuation on:
![image](https://user-images.githubusercontent.com/10785634/110171608-ef1e3480-7e36-11eb-9592-d85c0324e82d.png)
isDistanceAttenuation off:
![image](https://user-images.githubusercontent.com/10785634/110171628-f6ddd900-7e36-11eb-8ffb-c3789c18f1a1.png)

Reply:
Then I think the property should be called `distanceAttenuation`.

Reply:
OK. Shall I change all variables prefixed with `is`: `isOtherMeshes`, `isGroundReflector`, `isPerspectiveCamera`, `isFresnel`, `isDistanceAttenuation`, `isOtherMeshes`, `isBouncing`, `isInfiniteThick`, `isBlur`, `isSelecctive` to their none-`is`-prefixed version?

Reply:
I looked at several examples with `gui`, found really no setting name prefixed with 'is', and feel need ensure consistency, so I renamed all words prefixed with `is`.
In addition, renamed all `defines` to uppercase.